data/virt_autotest/virt-tests.conf

```diff
@@ -0,0 +1,136 @@
+[vmi_lifecycle_test]
```

Comment:
Suggest to rename the virt-tests.conf to sth like "kubevirt-virt-tests.conf".

Reply:
OK, renamed the config file to "kubevirt-tests.conf".